default/python/AI/AIDependencies.py

```diff
@@ -42,12 +42,9 @@
 # <editor-fold desc="Basic Game Rules">
 # TODO look into (enabling) simply retrieving the below values via UserString
 INDUSTRY_PER_POP = 0.2
-RESEARCH_PER_POP = 0.2
```

Comment:
This shouldn't be unused, there is actually a typo (or a c&p error) in colonization.calculate_research.

Reply:
I'll check and fix it in separate PR.

Reply:
https://github.com/freeorion/freeorion/pull/4245